sdk/core/core/src/az_json_read.c

```diff
@@ -0,0 +1,376 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_json_read.h>
+
+#include <az_digit.h>
+
+#include <math.h>
+
+inline bool az_json_is_white_space(char const c) {
+  switch (c) {
+    case ' ': case '\t': case '\n': case '\r':
+      return true;
```

Comment:
separate lines for the case statements.

also: good, you want a custom isspace instead of the standard one because locales.

Reply:
Guidelines strongly suggest to use clang-format. I recommend start using it using. I don't remember whether we agreed to use LLVM as the base or Microsoft as the base. Here is an online demonstrator: https://zed0.co.uk/clang-format-configurator/ . Adding .clang-format file allows to control the formatting all of the files at that level and all subdirectories